lib/list.c

```diff
@@ -51,3 +51,25 @@ void lib_listRemove(void **list, void *t, size_t noff, size_t poff)
 	*((addr_t *)(t + noff)) = NULL;
 	*((addr_t *)(t + poff)) = NULL;
 }
+
+
+int lib_listBelongs(void **list, void *t, size_t noff, size_t poff)
```

Comment:
why does this function take poffs?

Reply:
To keep the API the same as the rest of these functions. I'd rather keep it as is